README.md

```diff
@@ -6,92 +6,12 @@
 
 # ModUlar energy system Simulation Environment: MUSE
 
-## Installation
+## Installation instructions
 
-### Recommended way
+MUSE is available in `PyPI` and therefore it can be installed easily with `pip`. Detailed instructions on how to do that ensuring the right version of Python is used can be found in the Documentation.
 
```

Comment:
In the text of the PR it is indicated that the README needs updating once this is merge. I cannot update the README until that happens.

Reply:
Oh, OK, so I misunderstood and the README would still have back again the info on the installation once the doc is updated.

Reply:
No, I think I was not clear either. The README will be short, as I have left it, but the link to the documentation in RdDs cannot be added until this is merged because there's nothing to point to (I can point to the documentation overall, but not to the specific sections of it because those have changed). Is it clearer?